js/singers.js

```diff
@@ -0,0 +1,66 @@
+  },
+  {
+    name: 'Martin Valverde',
+    picture: 'images/sin-valverde.png',
+    pictureAlt: 'Valverde Singer photo',
+    txtFeatures: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit.',
+    txtExperience: 'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Possimus perspiciatis a sunt ullam deleniti! Pariatur temporibus cum voluptatem officiis quidem! Consectetur pariatur possimus perferendis ratione id rem, iure alias eligendi.',
+  },
+  {
+    name: 'Athenas María Venica',
+    picture: 'images/sing-athenas.png',
+    pictureAlt: 'Athenas Singer photo',
+    txtFeatures: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit.',
+    txtExperience: 'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Possimus perspiciatis a sunt ullam deleniti! Pariatur temporibus cum voluptatem officiis quidem! Consectetur pariatur possimus perferendis ratione id rem, iure alias eligendi.',
+  },
+  {
+    name: 'Hna. Glenda Hernández',
+    picture: 'images/sing-glenda.png',
+    pictureAlt: 'Hna. Glenda Singer photo',
+    txtFeatures: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit.',
+    txtExperience: 'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Possimus perspiciatis a sunt ullam deleniti! Pariatur temporibus cum voluptatem officiis quidem! Consectetur pariatur possimus perferendis ratione id rem, iure alias eligendi.',
+  },
+  {
+    name: 'Yuli & Josh',
+    picture: 'images/sing-JulyJosh.png',
+    pictureAlt: 'Yuli & Josh Singers photo',
+    txtFeatures: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit.',
+    txtExperience: 'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Possimus perspiciatis a sunt ullam deleniti! Pariatur temporibus cum voluptatem officiis quidem! Consectetur pariatur possimus perferendis ratione id rem, iure alias eligendi.',
+  },
+  {
+    name: 'Pbro. Rob Galea',
+    picture: 'images/sing-pGalea.png',
+    pictureAlt: 'Pbro. Rob Galea Singer photo',
+    txtFeatures: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit.',
+    txtExperience: 'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Possimus perspiciatis a sunt ullam deleniti! Pariatur temporibus cum voluptatem officiis quidem! Consectetur pariatur possimus perferendis ratione id rem, iure alias eligendi.',
+  },
+];
+
+const singerContainer = document.querySelector('.singers-cards');
+let singersHtml = '';
+
+singersArray.forEach((card, index) => {
+  singersHtml += '<div class="card-singer';
+  if (index > 1) {
+    singersHtml += ' to-hide';
+  }
+  singersHtml += `">
+    <img src="${card.picture}" alt="${card.pictureAlt}" class="singer">
+    <div class="singer-description">
+      <h3>${card.name}</h3>                
+      <p class="fetures">${card.txtFeatures}</p>
+      <hr class="small-gray">
+      <p class="singer-experience">${card.txtExperience}</p>
+    </div>
```

Comment:
- [ ] Great job for adding the requirements by the previous code reviewer for adding the button for see more but I think you made something wrong so it happens an error now since when I press on see more but I can't see the rest of the list so please fix it to work as expected 😊🎉🤩💯🚀



https://user-images.githubusercontent.com/58553711/227660712-50bac6df-7335-4b1b-b33f-ed9b74e95ae5.mp4